ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusTextMessage.qml

```diff
@@ -47,6 +47,11 @@ Item {
             isQuote = (formattedMessage.startsWith("<blockquote>") && formattedMessage.endsWith("</blockquote>"));
 
             if (root.isEdited) {
+                if (root.messageDetails.contentType === StatusMessage.ContentType.BridgeMessage) {
+                    // message from bridge does not have any tags, we need to add them here to format correctly
+                    formattedMessage = "<p>"+root.messageDetails.messageText+"</p>"
```

Comment:
Normal messages have parsed_text in database and there is an info about html tags.
Bridge messages does not have it. They are plain texts. I need to add <p> here in order to make make the correct formatting below.